Project/Project/Pages/PrizeDetails.cshtml

```diff
@@ -1,13 +1,36 @@
 ﻿@page
 @model Project.Pages.DetailsModel
 @{
+    var Year = ViewData["Year"];
+    var Category = ViewData["Category"];
+    var Share = ViewData["Share"];
+    var Motivation = ViewData["Motivation"];
 }
-
 <h1>Prize Details for @Html.DisplayFor(model => model.NobelLaureates.Laureates[0].Firstname) @Html.DisplayFor(model => model.NobelLaureates.Laureates[0].Surname)</h1>
 
 
 <hr />
 
+<h3>Prizes Display</h3>
```

Comment:
I was trying to add this new page but I don't know how to extract the right value from JSON data.
But you get what I mean.